src/ConEmu/FontMgr.cpp

```diff
@@ -1117,10 +1117,11 @@ void CFontMgr::InitFont(LPCWSTR asFontName/*=NULL*/, int anFontHeight/*=-1*/, in
 		}
 	}
 
-	CreateFontGroup(LogFont);
-
-	//2009-06-07 Реальный размер созданного шрифта мог измениться
-	SaveFontSizes((mn_AutoFontWidth == -1), false);
```

Comment:
This is initialization procedure. It's inappropriate to skip any functions without terminating the rest of startup procedure.